index.html

```diff
@@ -135,6 +151,17 @@
 						[coords[2], coords[3]]
 					];
 					map.fitBounds(bbox, options={padding: 10, duration: 5000});
+					if (coords[4] === "whole state of Texas") { // if we're zooming out to the whole state again
+						showHideLayer('texas-school-districts-poly', markerName='', showOnly=false, hideOnly=true);
+						showHideLayer('texas-school-districts-lines', markerName='', showOnly=false, hideOnly=true);
+					} else {
```

Comment:
...and if zooming in, we make the mask layer visible and set a filter to cut the zoomed-to district out of the mask.